ONSdigital/eq-author-app
# EAR-611 Add shortcode & title to logic rule

## Additional info
- [Ticket](https://collaborate2.ons.gov.uk/jira/browse/EAR-611) 

## What is the context of this PR?
This ticket adds, if present, the title and short code of a question to give context to the logic rules for skip and routing rules. 
A tooltip has also been wrapped around the text to display the full title when it has been truncated.

The shortcode and question title are added to the content picker select component.

#### Criteria
- [x] Add new text if present
- [x] Add tooltip
- [x] Add tests to support new text

### How to review
- Get setup as normal
- Make a four/five questions each with an answer
- Add a routing rule
- _**Should display `Select an answer`**_
- Choose an answer
- _**Should display the question `title`**_
- Go to that question and add a `shortcode`
- _**Should display `shortcode` in the routing rule now**_
- Check the same for skip conditions
- Go back to the question title and make it really long
- _**Should be truncated and have tooltip**_
- Make the title shorter but long enough that when you resize the browser the title truncates
- _**Should have a tooltip**_
- Make browser fullwidth
- _**Should not have a tool tip anymore**_

#### Write this up tomorrow
